revolv/project/views.py

```diff
@@ -108,6 +114,13 @@ def form_valid(self, form):
             messages.success(self.request, '$' + str(repayment_amount) + ' repaid by ' + project.org_name)
         return redirect(self.get_success_url())
 
+    # pass in Project Categories and Maps API key
+    def get_context_data(self, **kwargs):
+        context = super(ReviewProjectView, self).get_context_data(**kwargs)
+        context['GOOGLEMAPS_API_KEY'] = settings.GOOGLEMAPS_API_KEY
+        context['categories'] = [str(category.title) for category in self.get_object().category_set.all()]
```

Comment:
That was b/c it was in unicode, but yeah, I guess it's fine leaving it in unicode.